demo/src/screens/select/multiselect.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import createClass from 'create-react-class';
+import PropTypes from 'prop-types';
+import { FormGroup, Checkbox, Select } from '../../../../src/components';
+
+const FLAVOURS = [
+	{ label: 'Chocolate', value: 'chocolate' },
+	{ label: 'Vanilla', value: 'vanilla' },
+	{ label: 'Strawberry', value: 'strawberry' },
+	{ label: 'Caramel', value: 'caramel' },
+	{ label: 'Cookies and Cream', value: 'cookiescream' },
+	{ label: 'Peppermint', value: 'peppermint' },
+];
+
+const WHY_WOULD_YOU = [
+	{ label: 'Chocolate (are you crazy?)', value: 'chocolate', disabled: true },
+].concat(FLAVOURS.slice(1));
+
+var MultiSelectField = createClass({
```

Comment:
is there a reason this is using createClass instead of es6 class extension? same for inputselect.

if its just for demo its not a big deal, just found it odd.

Reply:
The createClass was from the react-select demo code.  I'll just leave for now.